src/main/java/hudson/remoting/RemoteClassLoader.java

```diff
@@ -887,6 +894,12 @@ public interface IClassLoader {
          */
         @NonNull
         ResourceFile[] getResources2(String name) throws IOException;
+
+        /**
+         * Name of the classLoader
+         * @since 3242
```

Comment:
assumes this will be the next thing released

Reply:
I do not think we bother with `@since` in CD-versioned components, though it is possible.